cvat-canvas3d/src/typescript/canvas3dView.ts

```diff
@@ -0,0 +1,213 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Canvas3dController } from './canvas3dController';
+import { Listener, Master } from './master';
+
+import consts from './consts';
+import * as THREE from 'three';
+import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader.js';
+
+import { Canvas3dModel, UpdateReasons, ActiveElement, Mode, Configuration } from './canvas3dModel';
+
+export interface Canvas3dView {
+    html(): HTMLDivElement;
+    render(): any;
+}
+
+export class Canvas3dViewImpl implements Canvas3dView, Listener {
+    private loadingAnimation: SVGSVGElement;
+    private text: SVGSVGElement;
+    private adoptedText: SVG.Container;
+    private background: HTMLCanvasElement;
+    private bitmap: HTMLCanvasElement;
+    private grid: SVGSVGElement;
+    private content: SVGSVGElement;
+    private attachmentBoard: HTMLDivElement;
+    private adoptedContent: SVG.Container;
+    private canvas: HTMLDivElement;
+    private gridPath: SVGPathElement;
+    private gridPattern: SVGPatternElement;
+    private controller: Canvas3dController;
+    private svgShapes: Record<number, SVG.Shape>;
+    private svgTexts: Record<number, SVG.Text>;
+    private drawnIssueRegions: Record<number, SVG.Shape>;
+    private activeElement: ActiveElement;
+    private configuration: Configuration;
+    private renderer: any;
+    private scene: any;
+    private camera: any;
+    private canvas3dBackground: any;
+    private serviceFlags: {
+        drawHidden: Record<number, boolean>;
+    };
+
+    private set mode(value: Mode) {
+        this.controller.mode = value;
+    }
+
+    private get mode(): Mode {
+        return this.controller.mode;
+    }
+
+    private stateIsLocked(state: any): boolean {
+        const { configuration } = this.controller;
```

Comment:
Property 'configuration' does not exist on type 'Canvas3dController'.ts(2339)

There are a lot of static linter issues below. Please, fix them.

Reply:
Resolved